bitnami/valkey/README.md

```diff
@@ -0,0 +1,1048 @@
+<!--- app-name: Valkey -->
+
+# Bitnami package for Valkey
+
+Valkey is an open source, advanced key-value store. It is often referred to as a data structure server since keys can contain strings, hashes, lists, sets and sorted sets.
+
+[Overview of Valkey](http://valkey.io)
+
+Disclaimer: Valkey is a registered trademark of Valkey Ltd. Any rights therein are reserved to Valkey Ltd. Any use by Bitnami is for referential purposes only and does not indicate any sponsorship, endorsement, or affiliation between Valkey Ltd.
+
+## TL;DR
+
+```console
+helm install my-release oci://registry-1.docker.io/bitnamicharts/valkey
+```
+
+Looking to use Valkey in production? Try [VMware Tanzu Application Catalog](https://bitnami.com/enterprise), the enterprise edition of Bitnami Application Catalog.
+
+## Introduction
+
+This chart bootstraps a [Valkey](https://github.com/bitnami/containers/tree/main/bitnami/valkey) deployment on a [Kubernetes](https://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+Bitnami charts can be used with [Kubeapps](https://kubeapps.dev/) for deployment and management of Helm Charts in clusters.
+
+### Choose between Valkey Helm Chart and Valkey Cluster Helm Chart
+
+You can choose any of the two Valkey Helm charts for deploying a Valkey cluster.
+
+1. [Valkey Helm Chart](https://github.com/bitnami/charts/tree/main/bitnami/valkey) will deploy a master-replica cluster, with the [option](https://github.com/bitnami/charts/tree/main/bitnami/valkey#valkey-sentinel-configuration-parameters) of enabling using Valkey Sentinel.
+2. [Valkey Cluster Helm Chart](https://github.com/bitnami/charts/tree/main/bitnami/valkey-cluster) will deploy a Valkey Cluster topology with sharding.
+
+The main features of each chart are the following:
+
+| Valkey                                     | Valkey Cluster                                             |
+|--------------------------------------------------------|------------------------------------------------------------------------|
+| Supports multiple databases                            | Supports only one database. Better if you have a big dataset           |
+| Single write point (single master)                     | Multiple write points (multiple masters)                               |
+| ![Valkey Topology](img/valkey-topology.png) | ![Valkey Cluster Topology](img/valkey-cluster-topology.png) |
```

Comment:
For now we only have regular Valkey, this can be removed

Reply:
Sure.